### ISSUE TITLE
[Debate] Documentation organization

### ISSUE THREAD

**Issue by francocipollone** [Ekumen-OS/andino_docs#2]
### Summary

This issue is meant to be used as discussion point to define the documentation organization hosted at: https://andino.readthedocs.io/en/latest/package_summary/andino_description.html

No matter what we have right now, or what docs we have in the repositories. Let's think the best way we could provide the docs to users. (Mostly to new ones).
Let's think what we want to have online, later on we see how we adapt the current packages' info

### Documentation organization
I will keep this updated:

```
1. Overview
  1.1 Andino Ecosystem
  1.2 Motivation

2. Building your robot
  2.1 Bill Of Materials
  2.2 Connections Diagram
  2.3 Assembly Process
  2.4 Calibrate your andino
  2.5 First steps

3. Getting started
  3.1 Connect to your andino
  3.2 Bringup Andino 
  3.3 Visualize sensor data using RViz
  3.4 Teleoperate your robot.

4. Advanced
  4.1 Use andino for mapping (SLAM)
  4.2 Navigate with andino.

5. Simulation

6. Andino ecosystem
  6.1 Main packages
  6.2 Related projects

7. Community
   7.1 How to contribute
   7.2 Discord   
```

**Comment by francocipollone** [comment]
CC @jballoffet @JesusSilvaUtrera

**Comment by JesusSilvaUtrera** [comment]
@francocipollone I agree on the division but I think in the `Simulation` part we can add info about the different simulators we are supporting and how to use each of them, and then point to the related projects if they want to see the code and stuff. Something like `5.1 Gazebo`, `5.2 Isaac Sim`...

Also, I wouldn't name 1.1 `Andino Ecosystem`, since that's the name of point 6. I would rather go for `Introduction` or maybe just `Andino`.

Apart from that, everything LGTM, thanks Franco!
